sensor/core/models.py

```diff
@@ -0,0 +1,47 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.db import models
+
+
+class GenericSensor(models.Model):
+    """Represents a sensor abstracting away the specifics of what it measures.
+
+    A sensor measures one kind of thing. A physical device might have
+    multiple logical sensors.
+    """
+
+    name = models.CharField(max_length=256)
+    model = models.CharField(max_length=128)
+
+    class Meta:
+        unique_together = [('name', 'model')]
+
+
+class Sensor(models.Model):
+    """Base class for specific sensor types."""
+
+    generic_sensor = models.OneToOneField(GenericSensor)
```

Comment:
Zupełny drobiazg, ale z różnych względów lepiej nazwy modeli, do których realizujemy relacje podawać jako stringi w formule 'app_label.ModelName', czyli tutaj mielibyśmy `generic_sensor = models.OneToOneField('sensor_core.GenericSensor')`. To załatwia nam m. in. problem potrzeby importowania modeli, jeżeli relacje robimy do modeli znajdujących się w innych appkach.